community/detectors/gitlab_cve_2021_22205/src/main/java/com/google/tsunami/plugins/detectors/cves/cve202122205/Cve202122205DetectorBootstrapModule.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Please update this to `2021`. Same for the rest of this PR.

Reply:
ok